src/Sylius/Behat/Page/Shop/HomePage.php

```diff
@@ -105,6 +105,16 @@ public function switchLocale($localeCode)
     /**
      * {@inheritdoc}
      */
+    public function countProductsItems()
+    {
+        $products = $this->getDocument()->findAll('css','.sylius-product-name');
+
+        return count($products);
```

Comment:
Side note: `getLatestProductsNames()` won't be difficult to implement, but will allow for more flexibility in the future.